packages/wrangler/src/deployment-bundle/entry.ts

```diff
@@ -57,10 +55,13 @@ export async function getEntry(
 				? path.resolve(config.site?.["entry-point"])
 				: // site.entry-point could be a directory
 					path.resolve(config.site?.["entry-point"], "index.js");
-		} else if (args.legacyAssets || config.legacy_assets) {
+		} else if (
+			args.legacyAssets ||
+			config.legacy_assets ||
+			args.experimentalAssets ||
+			config.experimental_assets
+		) {
 			file = path.resolve(getBasePath(), "templates/no-op-worker.js");
-		} else if (args.experimentalAssets || config.experimental_assets) {
-			file = path.resolve(getBasePath(), "templates/no-op-assets-worker.ts");
```

Comment:
we should prolly delete `templates/no-op-assets-worker.ts` if we're not going to use it anymore